src/customers/account-dashboard-stored-payment-methods.md

```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
 
 ![]({% link images/images-ee/customer-account-stored-payment-methods.png %}){: .zoom}
 _Stored Payment Methods_
+
+## Add stored payment method from checkout
```

Comment:
The Admin user guide is written for merchants, so describing storefront procedures are customer third-person.